src/archanaNJun20/BookMeetingRoom.java

```diff
@@ -0,0 +1,105 @@
+package archanaNJun20;
+
+public class BookMeetingRoom {
+
+	String meetingRoom;
+	double hours=0.0;
+	boolean nalandaFlag=false;
+	boolean takshashilaFlag=false;
+	boolean agraFortFlag=false;
+	boolean pratapGadhFlag=false;
+
+	void bookMeetingRoom(){
+		if(nalandaFlag==false){
+			meetingRoom="Nalanda";
+			hours=1;
+			displayInfo();
+			nalandaFlag=true;
+		}
+		else{
+			System.out.println("-->The meeting room Nalanda is already booked.");
+		}
+	}
+	void bookMeetingRoom(String meetingRoom){
+		if(meetingRoom!="Nalanda"){
```

Comment:
For String variable comparison we should write like this '!meetingRoom.equals("Nalanda")'

Reply:
The code is updated as per your suggestion. Thank you.